python/cudf_polars/cudf_polars/experimental/dask_serialize.py

```diff
@@ -29,12 +55,14 @@ def _(x: DataFrame | Column):
     @cuda_deserialize.register(DataFrame)
     def _(header, frames):
         with log_errors():
+            frames = frames_to_gpumemoryview(frames)
             assert len(frames) == 2
             return DataFrame.deserialize(header, tuple(frames))
 
     @cuda_deserialize.register(Column)
     def _(header, frames):
         with log_errors():
+            frames = frames_to_gpumemoryview(frames)
             assert len(frames) == 2
             return Column.deserialize(header, tuple(frames))
```

Comment:
I think this approach is more complicated than it needs to be, because we know exactly the structure of `frames`.

```suggestion
        with log_errors():
            metadata, gpudata = frames
            return DataFrame.deserialize(header, (metadata, plc.gpumemoryview(gpudata)))

    @cuda_deserialize.register(Column)
    def _(header, frames):
        with log_errors():
            metadata, gpudata = frames
            return Column.deserialize(header, (metadata, plc.gpumemoryview(gpudata)))
```

Reply:
I was aiming more for a potential "future-proof" solution, but I think it's fair to simplify that.